README.md

```diff
@@ -1,19 +1,19 @@
-# FLink Table Store
+# Flink Table Store
 
 Flink Table Store is a unified streaming and batch store for building dynamic tables on Apache Flink.
 
 Flink Table Store is developed under the umbrella of [Apache Flink](https://flink.apache.org/).
 
-## <a name="build"></a>Building the Project
+## Building the Project
 
 Run the `mvn clean package` command.
 
 Then you will find a JAR file that contains your application, plus any libraries that you may have added as dependencies to the application: `target/<artifact-id>-<version>.jar`.
 
-## <a name="contributing"></a>Contributing
+## Contributing
 
 You can learn more about how to contribute on the [Apache Flink website](https://flink.apache.org/contributing/how-to-contribute.html). For code contributions, please read carefully the [Contributing Code](https://flink.apache.org/contributing/contribute-code.html) section for an overview of ongoing community work.
 
-## <a name="license"></a>License
+## License
 
-The code in this repository is licensed under the [Apache Software License 2](LICENSE).
```

Comment:
`Apache Software License` is ASL. https://wiki.debian.org/DFSGLicenses#The_Apache_Software_License_.28ASL.29

Reply:
Just let us keep the readme same with our license.
![image](https://user-images.githubusercontent.com/5163645/158335643-9271f8a9-788f-409b-aec8-05d38e9d930d.png)

Reply:
well, I think ASL is fine to me, too. I revert the change.